src/__forks__/Relay.js

```diff
@@ -16,6 +16,15 @@
 const RelayDefaultNetworkLayer = require('RelayDefaultNetworkLayer');
 const RelayPublic = require('RelayPublic');
 const RelayStore = require('RelayStore');
+const warning = require('warning');
+
+if (__DEV__) {
+  warning(
+    Promise && Array.prototype.find,
```

Comment:
maybe change this to `typeof Promise === 'function'` to not throw if it's not defined.